content/docs/style-guide.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Style Guide
+linktitle: Style Guide
+weight: 7
+---
+
+Note: Non-Native Code(bindings) must use their original name style.
+
+### Packages
+Package names are denoted with snake case but single words are preferred.
```

Comment:
Show example of what this means. 

I'd also change this myself to remove the "single words" bit since it is not even true any more.